traitsui/testing/tester/wx/helpers.py

```diff
@@ -29,3 +91,73 @@ def mouse_click_button(control, delay):
         wx.wxEVT_COMMAND_BUTTON_CLICKED, control.GetId()
     )
     control.ProcessEvent(click_event)
+
+def mouse_click_text_ctrl(control, delay):
+    """ Performs a mouce click on a wxTextCtrl.
+
+    Parameters
+    ----------
+    control : wxObject
+        The wx Object to be clicked.
+    delay: int
+        Time delay (in ms) in which click will be performed.
+    """
+    if not control.IsEnabled():
+        return
+    if not control.HasFocus():
+        control.SetFocus()
+    wx.MilliSleep(delay)
+    click_event = wx.CommandEvent(
+        wx.wxEVT_COMMAND_LEFT_CLICK, control.GetId()
+    )
+    control.ProcessEvent(click_event)
+
+
+def key_click_text_ctrl(control, interaction, delay):
+    """ Performs simulated typing of a key on the given wxObject
+    after a delay.
+
+    Parameters
+    ----------
+    control : wxObject
```

Comment:
I think we only support `wx.TextCtrl` here, the `EmulateKeyPress` is not generally available I think. So let's be specific about it.